src/sentry/workflow_engine/processors/workflow.py

```diff
@@ -34,6 +36,71 @@ class WorkflowDataConditionGroupType(StrEnum):
     WORKFLOW_TRIGGER = "workflow_trigger"
 
 
+def get_actions_to_workflows(actions_to_fire: list[Action]) -> dict[int, Workflow]:
```

Comment:
curious about this method at a high level, we already have a list of the actions that have been filtered to what will trigger. do we need to do this lookup?